fabric-recipe-api-v1/src/main/java/net/fabricmc/fabric/api/recipe/v1/ingredient/CustomIngredientSerializer.java

```diff
@@ -54,6 +55,14 @@ static CustomIngredientSerializer<?> get(Identifier identifier) {
 	 */
 	Identifier getIdentifier();
 
+	/**
+	 * {@return the codec}.
```

Comment:
A small stylistic nitpick: should we enforce leaving out the period at the end of `{@return}`? Yarn primarily doesn't use punctuation with only 13 exceptions.

```suggestion
	 * {@return the codec}
```

Reply:
[Unfortunately, no.](https://github.com/checkstyle/checkstyle/issues/12600)

Reply:
In that case, should we change the Yarn convention to enforce the period? In any case, the style should be consistent disregarding any linter bugs, even if the exceptions aren't relevant to this pull request or repository specifically.

Reply:
I'd say Yarn's convention is the real one, and the one used in FAPI is just a workaround for a bug. The output is the same anyway iirc.

Reply:
No, the output in FAPI afaik produces double periods. Better than failing build.

Reply:
```java
public class Test {
    /** {@return a number}. */
    public int x() { return 0; }
}
```

![](https://github.com/FabricMC/fabric/assets/6596629/210b6db2-78a2-413c-8608-28cd6d5b592d)

Just tested (with JDK 21), I can confirm that. (I thought I confirmed the opposite once?) So in this case, we should absolutely not change Yarn's conventions.